examples/pod_namespace_watch.py

```diff
@@ -12,6 +12,13 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+"""
+Uses watch to print the stream of events from list namespaces and list pods.
```

Comment:
list_namespaces list_pods should have underscored in them , otherwise people may think of it as a Python list

Reply:
`list` and `watch` are coherent Kubernetes API concepts:
- https://kubernetes.io/docs/reference/using-api/api-concepts/
- https://github.com/kubernetes/community/blob/master/contributors/devel/sig-architecture/api-conventions.md#verbs-on-resources

I think it's okay for the description to stay high level and to avoid calling out specific method name